QuizApp/Scenes/QuizPage/ViewController/QuizViewController.swift

```diff
@@ -111,24 +111,14 @@ final class QuizViewController: UIViewController {
         return label
     }()
     
-    private let starImageView: UIImageView = {
-        let imageView = UIImageView()
-        imageView.image = .star.withRenderingMode(.alwaysOriginal)
-        imageView.contentMode = .scaleAspectFit
-        return imageView
-    }()
-    
     private let progressBar = ProgressView()
     
     // MARK: - Properties
-    private var viewModel: HomeViewModel
-    private var question: Question?
-    private var subject: Subject
+    private var viewModel: QuizViewModel
     
     // MARK: - Lifecycle
-    init(viewModel: HomeViewModel, subject: Subject) {
+    init(viewModel: QuizViewModel) {
         self.viewModel = viewModel
-        self.subject = subject
```

Comment:
I don't think that it's optimal for View to know about Subject or Question objects other than that looks good to me

Reply:
Absolutely, vc doesn't need to know about subject, I have changed it, I think you just viewed deletion line